templates/repo/diff/box.tmpl

```diff
@@ -15,11 +15,18 @@
 	<div>
 		<div class="diff-detail-box diff-box sticky gt-df gt-sb gt-ac gt-fw">
 			<div class="gt-df gt-ac gt-fw">
-				<a class="diff-toggle-file-tree-button muted gt-df gt-ac">
+				<button class="diff-toggle-file-tree-button tooltip gt-df gt-ac" data-show-text="{{.locale.Tr "repo.diff.show_file_tree"}}" data-hide-text="{{.locale.Tr "repo.diff.hide_file_tree"}}">
```

Comment:
Remove the "tooltip" class?

Reply:
Ah, it was already made obsolete, I missed that.

Reply:
Done